### PR TITLE
fix bug issue for #315

### PR DESCRIPTION
**Pull request by mharyam**
# Fix issue for #315 
# Fix issue for #316 

#### This PR is to fix issue #315 
and #316 

This affects the backend

<!-- Please link the issue you're trying to fix with this PR, if none then please create an issue first.

 Please include a summary of the change and which issue is fixed.
 Please also include relevant motivation and context.
 List any dependencies that are required for this change.

 If we didn't approve off or discuss your PR in Github issues or slack there's a high chance it will not be merged.

## This part affect the Back End

### CODE REVIEW

File: BackEndApp/authv1/views.py [hunk 1]
```diff
@@ -71,9 +71,13 @@ def register(request):
         message = "Some error occurred!! Please try again."
         status = 401
         token = None
-    return JsonResponse(
-        {"message": message, "username": username, "token": token}, status=status
-    )
+        return JsonResponse(
```
Comment (ADI10HERO) [inline BackEndApp/authv1/views.py:74]: Wouldn't it be better to just set username = None in the except block? Rest everything can remain same

Comment (RusherRG) [reply BackEndApp/authv1/views.py:74]: Yeah I think that might be better. Also do we actually need it? Because it's already included in the request JSON

**Comment by ADI10HERO** [comment]
Thank you for your contribution @mharyam. 

Apart from the code changes, I would like to highlight the following points..
1. We go for 1 PR per issue
2. PR title should be like `[Fixes #issueNumber] Pr title` 
3. I would also not remove the PR template :)

File: BackEndApp/authv1/views.py (hunk 1 above)
Comment (ADI10HERO) [reply BackEndApp/authv1/views.py:74]: Umm, you're right but I'd say let's not change the response altogether, might break something else in the front end 😅

**Comment by mharyam** [comment]
> Thank you for your contribution @mharyam.
> 
> Apart from the code changes, I would like to highlight the following points..
> 
> 1. We go for 1 PR per issue
> 2. PR title should be like `[Fixes #issueNumber] Pr title`
> 3. I would also not remove the PR template :)

Thank you for the review 

I will revert and push again

**Pull request closed without merging**